README.md

```diff
@@ -34,23 +53,122 @@ They are said to be "bound" to the unit they are currently operating on.
+ends the enzyme early, and an additional enzyme is determined from the remaining strand, decoding 
+until the end of the strand or another pun is reached. These enzymes will act sequentially on
+the strand, one at a time. The portion of a strand that encodes for a specific enzyme is known
+as a 'gene', so one strand may contain one or more genes that encode for the same number of enzymes. 
+Pun has no folding direction, and therefore does not contribute to the folding pattern of the enzyme, 
+as well as no additional effects on the enzyme past encoding. 
+ 
+#### cut
+
+Cuts the strand at the unit the enzyme is currently bound to.
+The cut is placed to the right of the bound unit.
+Cut applies to both the "active" strand and the "complementary" strand that
+is being generated from the copy mode.
+
+#### del 
+
+Deletes the base that the enzyme is attached to, and then the enzyme moves to the right. 
+Only pertains to the active strand, *not* the complementary strand. 
+
+#### swi
+
+Switches the enzyme from the "active" strand to the "complementary" strand. Since the complementary strand
+is represented upside-down above the active strand (see [cop](#cop)), switching strands seems to reverse the order 
+of the bases in the now active strand. If a switch occurs and there are no bases in the complementary strand above 
+where the enzyme is acting, the enzyme falls off and ends acting.  
+
+#### mvr
+
+Moves the enzyme one base to the right. If the enzyme moves off the end of the strand or into a gap with no bases, then it finishes executing
+even if it had more instructions left to execute. 
+
+#### mvl
+
+Moves the enzyme one base to the left. If the enzyme moves off the front of the strand or into a gap with no bases, then it finishes executing
+even if it had more instructions left to execute, same as in mvr. 
+
+#### cop
+
+Places the complement of the current base into the complementary strand, and enters "copy mode." Each base has its corresponding
+complement, known as the complementary base pairin (see [Definitions](#Definitions)). Copy mode remains in effect until an off 
+instruction is reached (or the end of the strand/enzyme). While in copy mode, any base that is "touched" by the enzyme will 
+have its complementary base placed above it in the complementary strand. When bases are copied into the complementary 
+strand, they are placed *upside-down* above the element copied, so that the strands are mirrored to each other. Copy mode applies 
+after a move, so that the movement off of a strand or into a gap between bases still preserves the behavior of the strand ending normally.  
+While in copy mode, additional cop instructions do nothing. 
+
+#### off
+
+Turns off copy mode. If copy mode is not enabled, then this does nothing. 
+
+#### ina
+
+Inserts an 'A' base into the position to the right of the enzyme, and if copy mode is on, then the base's complement (T) is placed above into
+the complementary strand. 
```

Comment:
Insert commands don't cause the enzyme to move to the right correct? So if you had `ina inc int ing` you would get `GTCA`? Since it always inserts to the right

Reply:
Yes that is correct. For some reason I was under the impression that all instructions moved the enzyme one unit to the right after they were done executing.

Reply:
Funny enough, since enzymes aren't required to always bind to the first binding base, you could have enzymes that primarily move from left to right instead of left to right.

It would be interesting to see if we could find a palindrome strand that would do the same thing backwards and forwards, depending on if it binds to the left or right side first.

Perhaps a 'select last binding base' flag would be interesting.